packages/generator/test/renderers/helpers.test.ts

```diff
@@ -98,7 +52,13 @@ describe("Renderer Helpers", () => {
       expect(models)
         .to.be.an("array")
         .with.lengthOf(1);
-      expect(models[0]).to.be.an.instanceOf(model.document.Document);
```

Comment:
This test should still be included, just change the expect to `models[0].document`.